tasks/internal/request-and-upload-signature/tests/test-request-and-upload-signature-batch-signer.yml

```diff
@@ -0,0 +1,170 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-request-and-upload-signature-batch-signer
+  annotations:
+    test/assert-task-failure: "run-task"
+spec:
+  description: |
+    Run the request-and-upload-signature task with batch signer.
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:3729a54989514da6a777579feabbd9b346c73551
```

Comment:
I see there is a complete inconsistency in the quay.io/konflux-ci/release-service-utils image version across your changed files.
Maybe it's worth taking care of the version match?
At least for the main tasks and related tests.
Ideally, both tasks (request-and-upload-signature and collect-simple-signing-params) should use the same version since they're related.